lib/run.js

```diff
@@ -32,10 +32,14 @@ function validateOpts(options) {
     
     validateNumberProp(options, 'rate');
     validateNumberProp(options, 'concurrent');
-    
-    // hack to enable concurrency mode,
-    // since forkmaster expects this to be defined
-    options.rate = options.rate || 0;
+
+    if (options.concurrent) {
+        delete options.rate;
+    }
+
+    if (options.rate < 0 || Math.floor(options.rate * (options.duration / 1000)) < 1) {
+        throw new Error('duration is too low');
```

Comment:
Is another run mode more reasonable in that case? If you can't get a test to run in your allotted time, maybe it makes more sense to say `grandma run count=1`.